requirements.txt

```diff
@@ -1,33 +1,38 @@
-alembic==1.5.4
```

Comment:
I'm seeing some deprecation warnings for how some of these packages get installed:

```
DEPRECATION: SQLAlchemy is being installed using the legacy 'setup.py install' method, 
because it does not have a 'pyproject.toml' and the 'wheel' package is not installed. 
pip 23.1 will enforce this behaviour change. A possible replacement is to enable 
the '--use-pep517' option. 
Discussion can be found at https://github.com/pypa/pip/issues/8559
```

I do have wheel globally available, but it doesn't seem to pick it up for the venv, and I'm not getting clear guidance on how to resolve this (manually installing wheel to the venv then installing from requirements seemed to work, but [this](https://stackoverflow.com/questions/69369669/python-3-venv-and-the-wheel-package) was pretty wishy washy about what to do). And so I'm not sure how best to address this for students.

Maybe we ignore for now. Thoughts?

Reply:
I'm inclined to ignore the warnings.  I tested an install of python@3.11 and verified seeing those Deprecation warnings. I ran flask and everything worked as expected. I also have `wheel` installed globally but didn't need to install it to the venv in order to run the app. So, I'm thinking we can just ignore the warnings until they either they go away (or if something actually breaks).